OvmfPkg/IntelTdx/README.md

```diff
@@ -62,10 +62,10 @@ cd /path/to/edk2
 source edksetup.sh
 
 ## without CC_MEASUREMENT enabled
-build -p OvmfPkg/OvmfPkgX64.dsc -a X64 -t GCC5 -b RELEASE
+build -p OvmfPkg/OvmfPkgX64.dsc -a X64 -t GCC5 -D CC_MEASUREMENT_ENABLE=FALSE -b RELEASE
 
 ## CC_MEASUREMENT enabled
-build -p OvmfPkg/OvmfPkgX64.dsc -a X64 -t GCC5 -D CC_MEASUREMENT_ENABLE=TRUE -b RELEASE
+build -p OvmfPkg/OvmfPkgX64.dsc -a X64 -t GCC5 -b RELEASE

```

Comment:
I think it is still valid to set `-D CC_MEASUREMENT_ENABLE=TRUE`

Reply:
Agree. Will updated in next version.